package.json

```diff
@@ -64,30 +64,33 @@
   },
   "devDependencies": {
     "babel": "^6.23.0",
-    "babel-core": "^6.23.1",
-    "babel-eslint": "^7.1.1",
-    "babel-jest": "^19.0.0",
-    "babel-loader": "^6.3.2",
-    "babel-plugin-istanbul": "^4.1.4",
-    "babel-preset-env": "^1.6.0",
-    "coveralls": "^2.13.1",
-    "eslint": "^3.16.1",
-    "eslint-config-ship-components": "^1.4.2",
-    "eslint-loader": "^1.6.3",
-    "eslint-plugin-react": "^7.0.0",
-    "extract-text-webpack-plugin": "^2.0.0",
+    "babel-core": "^6.26.0",
+    "babel-eslint": "^8.1.2",
+    "babel-jest": "^22.0.4",
+    "babel-loader": "^7.1.2",
+    "babel-plugin-istanbul": "^4.1.5",
+    "babel-preset-env": "^1.6.1",
+    "coveralls": "^3.0.0",
+    "eslint": "^4.14.0",
+    "eslint-config-ship-components": "^1.5.1",
+    "eslint-loader": "^1.9.0",
+    "eslint-plugin-react": "^7.5.1",
+    "extract-text-webpack-plugin": "^3.0.2",
     "grunt": "^1.0.1",
     "grunt-cli": "^1.2.0",
-    "grunt-contrib-clean": "^1.0.0",
+    "grunt-contrib-clean": "^1.1.0",
     "grunt-coveralls": "^1.0.1",
-    "grunt-eslint": "^19.0.0",
-    "grunt-webpack": "^2.0.1",
-    "husky": "^0.13.4",
-    "jest": "^20.0.1",
-    "istanbul-lib-instrument": "^1.7.3",
-    "jest-cli": "^19.0.2",
-    "lint-staged": "^3.6.1",
+    "grunt-eslint": "^20.1.0",
+    "grunt-webpack": "^3.0.2",
+    "husky": "^0.14.3",
+    "istanbul-lib-instrument": "^1.9.1",
+    "jest": "^22.0.4",
+    "jest-cli": "^22.0.4",
+    "lint-staged": "^6.0.0",
     "strict-loader": "^1.1.0",
-    "webpack": "^2.2.1"
+    "webpack": "^3.10.0"
+  },
+  "dependencies": {
+    "trevor": "^2.3.0"
```

Comment:
This should be a dev dependency. We don't need to install this in all of the apps.

Reply:
Fixed.